.github/workflows/web.yaml

```diff
@@ -11,6 +11,11 @@ on:
         required: false
         type: string
         default: "Dockerfile"
+      deployment_type:
```

Comment:
Added this to differentiate between an integration which was triggered manually or automatically by merging a branch to master